### PR TITLE
added ega, dbgap accessions. Fixes #1336

### LINKED ISSUE #1336: Enable capture of portal links and SCEA, EGA & dbGap accessions
**For which schema is a change/update being suggested?**

project.json

**What should the change/update be?**

I would like to add at least two optional fields to the project_core.json schema

#### ega accession
* Field name: ega_accessions
* Field description: European Genome Phenome Archive (EGA) dataset or study accession. Study accession preferred.
* Field type: list of strings
* Required: No
* Examples: EGAS00000000001; EGAD00000000002
* pattern: `EGA[DS][0-9]{11}`

#### dbgap accession
* Field name: dbgap_accessions
* Field description: database of Genotypes and Phenotypes (dbGaP) study accession.
* Field type: list of strings
* Required: No
* Examples: phs001997.v1.p1; phs001836
* pattern: `phs[0-9]{6}(\.v[0-9]{1})?(\.p[0-9])?` (this regex should be double checked)

#### scea accession
* Field name: scea_accessions
* Field description: Single cell expression atlas accession.
* Field type: list of strings
* Required: No
* Examples: E-MTAB-6678;E-GEOD-150728
* pattern: `E-[A-Z]{4}-[0-9]+` (this regex should be double checked)

#### contributor portal links
* Field name: portal_links
* Field description: Links to a contributor created web-based portal with visualisations for this project.
* Field type: list of strings
* type: string
* format: uri
* Required: No
* Examples: https://www.heartcellatlas.org/
* pattern: `^https?://.+` (this regex should be double checked)

This update constitutes a minor change to the schema(s) it affects.

**Why is the change requested?**

To support additional links to managed access archives when that is where the data is available and contributor portal links.

### PR DESCRIPTION
**Pull request by mshadbolt**
### Release notes

For type/project schema:
- added ega_accessions field to capture EGA Study and Dataset accessions
- added dbgap_accessions field to capture dbgap study accessions
 

### Reviews requested

- Need 2 Reviewers to approve because this is a minor update
- Last Reviewer to review/approve, merge changes by Tuesday 23rd of March

### CODE REVIEW

**Comment by lauraclarke** [comment]
@mshadbolt why the change of plan? is there a reason not to go for the identifiers module?

**Comment by mshadbolt** [comment]
yes, because it will be a major change and require a migration of all current projects under the new metadata regime. We decided it would be best to stick with the simplest plan we need to support capturing managed access accessions for now.

**Comment by lauraclarke** [comment]
Ah, that was an outcome that clearly wasn't communicated. Hannes said he would be happy to support multiple schema versions via the metadata API at least to start with and my understanding if that while this is a major schema version change it doesn't impact backwards compatibility of any existing fields so the impact of the change shouldn't be big

Are there other reasons not to do this?

**Comment by mshadbolt** [comment]
I was told we wanted to make the change like this to simplify things so I made a new PR. I am not aware of what Hannes did or didn't say since I am unable to attend the majority of meetings about this.

**Comment by ESapenaVentura** [comment]
We can come back to the modules once we have figured out a way to migrate (I want to avoid major changes until we have at least identified requirements to record). For now, I feel like this is a good change

File: json_schema/update_log.csv [hunk 1]
```diff
@@ -1 +1,2 @@
 Schema,Change type,Change message,Version,Date
+type/project,minor,"Added ega_accessions and dbgap_accessions. Fixes #1336.",,
```
Comment (ESapenaVentura) [inline json_schema/update_log.csv:2]: `type/project/project`

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
LGTM 👍

**Comment by Wkt8** [comment]
Have made enrique's change for type/project/project.json as the schema

**Review by Wkt8 — APPROVED** [review]

**Pull request merged**